pkg/clientcert/certificate_test.go

```diff
@@ -161,10 +161,24 @@ func TestIsCertificateValid(t *testing.T) {
 
 func TestGetCertValidityPeriod(t *testing.T) {
 	certs := []byte{}
-	certs = append(certs, testinghelpers.NewTestCert("cluster0", 10*time.Second).Cert...)
+	firstCert := testinghelpers.NewTestCert("cluster0", 10*time.Second).Cert
+	certs = append(certs, firstCert...)
 	secondCert := testinghelpers.NewTestCert("cluster0", 5*time.Second).Cert
 	certs = append(certs, secondCert...)
-	expectedCerts, _ := certutil.ParseCertsPEM(secondCert)
+
+	firstCerts,_:= certutil.ParseCertsPEM(firstCert)
+	secondCerts,_ := certutil.ParseCertsPEM(secondCert)
```

Comment:
keep a space between `secondCerts` and  `_`

Reply:
Done

Reply:
should be
```
firstCerts, _ := certutil.ParseCertsPEM(firstCert)
secondCert, _ := certutil.ParseCertsPEM(secondCert)
```